packages/panzer/disc-fe/test/dofmngr_test/UnitTest_UniqueGlobalIndexer.cpp

```diff
@@ -246,6 +246,36 @@ void UniqueGlobalIndexer::getOwnedIndices(std::vector<int> & indices) const
    }
 }
 
```

Comment:
All the `UniqueGlobalIndexer`s such that they now have:
* `getOwnedIndices()`
* `getGhostedIndices()`
* `getOwnedAndGhostedIndices()`
* `getNumOwned()`
* `getNumGhosted()`
* `getNumOwnedAndGhosted()`